tools/topology/topology2/platform/intel/dmic-generic.conf

```diff
@@ -140,7 +140,7 @@ IncludeByKey.PASSTHROUGH {
 						}
 					]
 					Object.Control.mixer.1 {
-						name '$DMIC0_PCM_NAME Capture Volume'
+						name 'Dmic0 Capture Volume'
```

Comment:
So this is for sure correct for the hda-generic case and the SDW tplgs we have shipped in SOF2.8 did not have dmic so that is good as well.

So this leaves the Chrome sof-mtl-max98357a-rt5682*tplg .that have DMICs and are using this definition. FYI @yongzhi1 @jairaj-arava , this will change the ALSA mixer control name for dmic volume control. This aligns the change to existing UCM files in https://github.com/alsa-project/alsa-ucm-conf/blob/master/ucm2/Intel/sof-hda-dsp/HiFi.conf